editor/js/Menubar.File.js

```diff
@@ -123,6 +123,64 @@ Menubar.File = function ( editor ) {
 	} );
 	options.add( option );
 
+	// Export Material
+
+	var option = new UI.Row();
+	option.setClass( 'option' );
+	option.setTextContent( strings.getKey( 'menubar/file/export/material' ) );
+	option.onClick( function () {
+
+		var object = editor.selected;
+
+		if ( object === null ) {
+
+			alert( 'No object selected.' );
+			return;
+
+		}
+
+		var material = object.material;
+
+		if ( material === undefined ) {
+
+			alert( 'The selected object doesn\'t have material.' );
+			return;
+
+		}
+
+		if(!(material instanceof Array)){
+			material = [material];
+		}
+
+		for(var i = 0; i < material.length; i++) {
+
+			var output = material[i].toJSON();
+			
+			try {
+				
+				output = JSON.stringify( output, parseNumber, '\t' );
+				output = output.replace( /[\n\t]+([\d\.e\-\[\]]+)/g, '$1' );
+				
+			} catch ( e ) {
+				
+				output = JSON.stringify( output );
+				
+			}
+			
+			if(material.length === 1) {
```

Comment:
There are some code style issues in your PR. Best approach is to work with a correctly configured editor/IDE in order to autodetect such stuff. Otherwise use https://zz85.github.io/mrdoobapproves/